source/renderer/app/components/splash/SplashNetworkITN.js

```diff
@@ -47,6 +48,7 @@ const messages = defineMessages({
 type Props = {
   onClose: Function,
   openExternalLink: Function,
+  isIncentivizedTestnetTheme?: boolean,
```

Comment:
@DeeJayElly please make this mandatory prop 🙏

Reply:
Done